.circleci/config.yml

```diff
@@ -428,6 +428,26 @@ jobs:
           paths:
             - .tox
 
+  influx:
+    docker:
+      - *test_runner
+      - image: influxdb:1.5-alpine
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+              - tox-cache-influx-{{ checksum "tox.ini" }}
+      - run: TOX_SKIP_DIST=False tox -e '{py27,py34,py35,py36}-influx{50,51,52}' --result-json /tmp/influx.results
```

Comment:
A couple of things here:
 - You can skip `TOX_SKIP_DIST=False` as we do not need a complete `dist->install->test` process here. We use this approach only for autopatch tests. You will see that tests will tests for the various envs will be much faster.
 - Based on the comments below, the command would then be: `tox -e 'influx_contrib-{py27,py34,py35,py36}-influx{50,51,52}' --result-json /tmp/influx.results`